docs/performance.md

```diff
@@ -9,23 +9,23 @@ This guide is intended to teach you some basics to help you to [troubleshoot per
+![Performance Monitor screenshot](/docs/assets/PerfUtil.png)
 
 ### JS frame rate (JavaScript thread)
 
-For most React Native applications, your business logic will run on the JavaScript thread. This is where your React application lives, API calls are made, touch events are processed, etc... Updates to native-backed views are batched and sent over to the native side at the end of each iteration of the event loop, before the frame deadline (if all goes well). If the JavaScript thread is unresponsive for a frame, it will be considered a dropped frame. For example, if you were to call `this.setState` on the root component of a complex application and it resulted in re-rendering computationally expensive component subtrees, it's conceivable that this might take 200ms and result in 12 frames being dropped. Any animations controlled by JavaScript would appear to freeze during that time. If anything takes longer than 100ms, the user will feel it.
+For most React Native applications, your business logic will run on the JavaScript thread. This is where your React application lives, API calls are made, touch events are processed, and more. Updates to native-backed views are batched and sent over to the native side at the end of each iteration of the event loop, before the frame deadline (if all goes well). If the JavaScript thread is unresponsive for a frame, it will be considered a dropped frame. For example, if you were to set a new state on the root component of a complex application and it resulted in re-rendering computationally expensive component subtrees, it's conceivable that this might take 200ms and result in 12 frames being dropped. Any animations controlled by JavaScript would appear to freeze during that time. If enough frames are dropped, the user will feel it.
 
-This often happens during `Navigator` transitions: when you push a new route, the JavaScript thread needs to render all of the components necessary for the scene in order to send over the proper commands to the native side to create the backing views. It's common for the work being done here to take a few frames and cause [jank](https://jankfree.org/) because the transition is controlled by the JavaScript thread. Sometimes components will do additional work on `componentDidMount`, which might result in a second stutter in the transition.
+This often happens during navigation transitions when pushing a new route. For example, with [React Navigation's JavaScript-based stack navigator](https://reactnavigation.org/docs/stack-navigator/), the JavaScript thread needs to render all components for the new screen and send commands to the native side to create backing views. This can take multiple frames and cause [jank](https://jankfree.org/) because the transition animations are controlled by the JavaScript thread. Sometimes components will do additional work inside various hooks that run on mount which might result in additional stutters during the transition.
 
```

Comment:
The reasoning is not quite accurate since animations still run natively with the native driver when using JS stack, not controlled by the JavaScript thread.

Reply:
@satya164 Do you have any suggestions for how to word this more accurately?

Reply:
It's a collection of a lot of different things like there not being native gesture + animation integration in react native, animations capped to 60 FPS, UI updates affecting animations, etc. But these are not quite related to animations running on JS thread/JS thread in general, which this section is about, so I'm not really sure what can be written here.

Reply:
I guess I can talk about `Animate` and `useNativeDriver` instead, here. But yeah, this is honestly so far out of date, perhaps we just remove it?